### PR TITLE
[openwebnet] added channel "function" for Central Unit

### PR DESCRIPTION
**Pull request by aconte80**
Enhance OpenWebNet binding to support thermoregulation with 4/99 zone central unit, e.g. BTicino 3550 model.

This PR adds the `function` channel  (allowed values HEATING/COOLING) to the binding. 
It is a readonly channel, the value can only be changed manually acting on the Central Unit.

### CODE REVIEW

File: bundles/org.openhab.binding.openwebnet/src/main/java/org/openhab/binding/openwebnet/internal/handler/OpenWebNetThermoregulationHandler.java [hunk 1]
```diff
@@ -280,18 +280,31 @@ protected void handleMessage(BaseOpenMessage msg) {
...
 
             if (msg.getWhat() == Thermoregulation.WhatThermo.REMOTE_CONTROL_DISABLED) {
                 updateCURemoteControlStatus(CU_REMOTE_CONTROL_DISABLED);
             } else if (msg.getWhat() == Thermoregulation.WhatThermo.REMOTE_CONTROL_ENABLED) {
                 updateCURemoteControlStatus(CU_REMOTE_CONTROL_ENABLED);
             } else if (msg.getWhat() == Thermoregulation.WhatThermo.BATTERY_KO) {
                 updateCUBatteryStatus(CU_BATTERY_KO);
+            } // must intercept all possibile WHATs (will be implemented soon)
+            else if (msg.getWhat() == Thermoregulation.WhatThermo.AT_LEAST_ONE_PROBE_OFF) {
+                logger.info("handleMessage() Ignoring unsupported WHAT {}. Frame={}", msg.getWhat(), msg);
```
Comment (lolodomo) [inline bundles/org.openhab.binding.openwebnet/src/main/java/org/openhab/binding/openwebnet/internal/handler/OpenWebNetThermoregulationHandler.java:295]: Please DEBUG level

**Review by lolodomo — CHANGES_REQUESTED** [review]

**Review by mvalla — COMMENTED** [review]
I have reviewed the changes and do not have any specific comment. LGTM.

**Review by lolodomo — APPROVED** [review]
LGTM

**Comment by mvalla** [comment]
> As a side note, this binding is still missing a full default translations file.

can you provide a reference/doc about this point and how to provide the translations file ?

**Comment by lolodomo** [comment]
This is only a mvn command to run. 
I will copy / paste the command later.

**Comment by lolodomo** [comment]
@mvalla : thge command to generate defaylt translation properties is: mvn org.openhab.core.tools:i18n-maven-plugin:3.2.0:generate-default-translations

**Pull request merged**